packages/design-system/src/components/Toggle/Toggle.interface.ts

```diff
@@ -1,16 +1,19 @@
 import { ClassName } from "@helpers/types";
 import { InputHTMLAttributes } from "react";
 
+export type label =
+  | { visibleLabel: string; ariaLabel?: string }
+  | { visibleLabel?: string; ariaLabel: string };
+
 export interface ToggleProps
   extends Omit<InputHTMLAttributes<HTMLInputElement>, "size" | "onChange">,
     ClassName {
   size: "sm" | "md";
-  labelText?: string;
+  label: label;
```

Comment:
```suggestion
  label: string;
```
Label, which is an object, is not very intuitive.